charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.19.3
+
+* Fix `dsdsocket` volumeMount for the `trace-agent` on linux deployment
```

Comment:
I think it would be worthwhile to also note here that this fixes mounting the `runtimesocket` for `trace-agent` on Windows.